sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment.py

```diff
@@ -573,3 +574,93 @@ def callback(pipeline_response, deserialized, _):
             cls=callback
         )
         assert res == "cls result"
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    def test_aspect_based_sentiment_analysis(self, client):
```

Comment:
Another test we could add is to turn on opinion mining with a sentence that gives us no results.
For example for the sentence: "today is a hot day"

Reply:
great suggestion, adding that to this PR. thanks!